setup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+
+from glob import glob
+from os.path import basename
+from os.path import splitext
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name='selectedtests',
```

Comment:
What do you recommend? I'd also probably have to rename it in the [technical design](https://docs.google.com/document/d/1azKrJr3babowhr6M8vs9cCTQOmLdUeyqmyVKp_ENSOw/edit), but I am open to changing it.

Reply:
I thin David and Jeff have more to say about naming than I do. I'll drop my request since I've not come up with a good name, and snake case is apparently against python standards.